### DISCUSSION TITLE
Not all reactions shown in the hub's feed?

### DISCUSSION THREAD

**Discussion by Flor-e** [alkem-io/alkemio#866] [Q&A]
I just added two cards on the explore page of this challenge: https://alkem.io/challengecentric/challenges/active-contributions/dashboard

I noticed that these contributions:
- Do not show in the feed on the hub's dashboard, only on the challenge dashboard.
- Do not trigger an e-mail that a new contribution is made (while I am a member of the challenge). 

Is that by design? I think a lot of interaction is now 'lost' because people are simply unaware of them.

That being said: sending out emails on every interaction on challenge-level might prove to be a bit too much. So maybe as a member you should only be directly informed if a new callout, opportunity or challenge is created and only sparingly when there are new reactions to a callout (for example: only when you have reacted to the callout yourself, or only once a week).

For hub/challenge-admins on the other hand: I do think they should have the option to always be informed by email when a new card is created on the explore page.

**Comment by techsmyth** [comment]
HI Flori, 

As always good observations :)

Yes at the moment the interactions are only shown within the direct context. I assume you would like to also see the challenge activity then on the Hub activity overview?

Re notification not made, there is a gap in that comments on a Discussion Call out do not trigger them - otherwise if you are a member it should be triggered. Can you give more details of the interaction that did not show up?

The visibility of activity and control over notifications are both topics that will need some good consideration near term. 
@SimoneZaza @larssondenise fyi.

Cheers, Neil

**Reply by Flor-e** [reply]
Hi Neil,

Yes, I would somehow like one place where I can see all the hub's activity. Because in a hub that has many challenges (like the challenge-centric hub) it is a lot of work to browse every single challenge/opportunity. I for example found out that there was a callout in the "Enabling Online" challenge. Even though I am not signed up as a member for this challenge, it was something I had an opinion about. 

Collecting all activity in one feed does however (in my opinion) mean that the feed should be more enriched (for example by using visual cues to quickly see what type of activity it is and maybe even if it is 'popular') and more social media-like, like we talked about earlier.

Regarding the notification that did not trigger. I created a canvas in the "Active contributions"-channel: https://alkem.io/challengecentric/challenges/active-contributions/explore This event did not trigger an e-mail notification. I initially thought that the 2 callout response-cards I created also didn't trigger anything, but it turned out it did. So it's only the canvasses.

**Reply by techsmyth** [reply]
Hi Flori, 

Yes indeed would be very useful to surface up all the activity in a particular Hub. In general the notion of activity will I think be coming up the priority, as besides the actual events one would also want to see where the "energy" is? 

I just checked, and while I thought we had covered Canvas being created notification with the Cards we had not. It is now also added to the backlog: https://github.com/alkem-io/notifications/issues/156 - I expect notifications in general to get picked up in the next sprint. 

Thanks for all the feedback, and sorry again for the slower repsonse...

Cheers, Neil
